src/main/java/gregtech/common/covers/CoverConveyor.java

```diff
@@ -469,7 +471,9 @@ public void readFromNBT(NBTTagCompound tagCompound) {
             this.isWorkingAllowed = tagCompound.getBoolean("WorkingAllowed");
         }
         if (tagCompound.hasKey("AllowManualIO")) {
-            this.allowManualImportExport = tagCompound.getBoolean("AllowManualIO");
+            this.manualImportExportMode = tagCompound.getBoolean("AllowManualIO")
+            ? ManualImportExportMode.FILTERED
+            : ManualImportExportMode.DISABLED;
         }
```

Comment:
This should be moved up just under LEGACY SAVE FORMAT SUPPORT as it specifically falls to that category.